### PR TITLE
Remote Elasticsearch cluster is not supported

### PR DESCRIPTION
**Pull request by nimarezainia**

### CODE REVIEW

File: docs/en/ingest-management/fleet/fleet-settings.asciidoc [hunk 1]
```diff
@@ -155,6 +155,8 @@ output is set in the agent policy.
 
 | When this setting is on, {agent}s use this output to send agent monitoring
 data if no other output is set in the agent policy.
+
+Sending monitoring data to a Remote Elasticsearch is currently not supported.
```
Comment (dedemorton) [inline docs/en/ingest-management/fleet/fleet-settings.asciidoc:159]:
```suggestion
Sending monitoring data to a remote {es} cluster is currently not supported.
```

**Comment by dedemorton** [comment]
@nimarezainia This one never got merged. Is this still true? Which versions do we need to backport it to?

**Review by dedemorton — APPROVED** [review]
LGTM

**Comment by bmorelli25** [comment]
@nimarezainia @dedemorton Should we merge this one or close it?

**Comment by dedemorton** [comment]
Merging...we can always back out the changes if needed.

**Comment by nimarezainia** [comment]
> @nimarezainia This one never got merged. Is this still true? Which versions do we need to backport it to?

yes still true and we don;t have a clear timeline on it. is there a way to say  backport all the way to 7.14?

**Comment by bmorelli25** [comment]
> yes still true and we don;t have a clear timeline on it. is there a way to say backport all the way to 7.14?

It looks like this config option only exists in 8.2 and newer (https://github.com/elastic/observability-docs/pull/1807)

**Comment by nimarezainia** [comment]
@bmorelli25 oh shoot yes thank you. The multiple outputs was on;y introduced then.

**Pull request merged**